app/helpers/application_helper.rb

```diff
@@ -31,7 +31,7 @@ def google_universal_analytics_json(ua_domain_name = nil)
     if SiteSetting.ga_universal_auto_link_domains.present?
       result[:allowLinker] = true
     end
-    result.to_json.html_safe
```

Comment:
hmm why was this removed?

Reply:
We are now rendering this json in a HTML attribute, so it has to be escaped (no longer html safe)

```html
<!-- html_safe -->
<meta data-json="{"key": "value"}">

<!-- without html_safe -->
<meta data-json="{&quot;key&quot;: &quot;value&quot;}">
```

Reply:
:+1: